src/components/messages_container/index.js

```diff
@@ -1,11 +1,13 @@
 import React, { Component } from 'react';
 import compose from 'recompose/compose';
+import { connect } from 'react-redux';
 import classnames from 'classnames/bind';
+import { withDetails } from '@/hoc';
 import Header from './header';
 import MessageInput from './input';
 import List from './list';
 import Typings from './typings';
-import { withDetails } from '@/hoc';
+import inputActions from './input/actions';
```

Comment:
```suggestion
import {setText, reset} from './input/actions';
```
так красивее потом использовать <3

Reply:
чет не работает так. undefined